Cargo.toml

```diff
@@ -30,7 +30,7 @@ rand_core = { version = "0.5", default-features = false, optional = true }
 serde_crate = { package = "serde", version = "1.0", default-features = false, optional = true }
 serde_bytes = { version = "0.11", optional = true }
 sha2 = { version = "0.9", default-features = false }
-zeroize = { version = "~1.3", default-features = false }
+zeroize = { version = "1", default-features = false }
```

Comment:
This is great, would love a release with this change!

Reply:
See dalek-cryptography/curve25519-dalek#405